jaxopt/_src/loss.py

```diff
@@ -57,7 +57,9 @@ def binary_logistic_loss(label: int, logit: float) -> float:
   # Softplus is the Fenchel conjugate of the Fermi-Dirac negentropy on [0, 1].
   # softplus = proba * logit - xlogx(proba) - xlogx(1 - proba),
   # where xlogx(proba) = proba * log(proba).
-  return softplus(logit) - label * logit
```

Comment:
Could you add this implementation to the tests and check that the new one recovers the same value?